VTEX - Orders API.json

```diff
@@ -797,14 +797,218 @@
+                                                                                },
+                                                                                "city": {
+                                                                                    "type": "string",
+                                                                                    "description": "Package current tracking city."
+                                                                                },
+                                                                                "state": {
+                                                                                    "type": "string",
+                                                                                    "description": "Package current tracking state in two-digit code format."
+                                                                                },
+                                                                                "description": {
+                                                                                    "type": "string",
+                                                                                    "description": "Package tracking description."
+                                                                                },
+                                                                                "createDate": {
+                                                                                    "type": "string",
+                                                                                    "description": "Package tracking creation date in [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DD hh:mm:ss[.nnnnnnn]+hh:mm`."
+                                                                                }                                                    
+                                                                            }    
+                                                                        }
+                                                                    }                                      
+                                                                }
+                                                            },
+                                                            "cfop": {
+                                                                "type": "string",
+                                                                "description": "Operations tax code, when applicable. For example, sending promotional gifts to customers.",
```

Comment:
Not sure I understand this description, especially since the example value is a random code 🤔 Is this valid only for Brazil [as determined by the government](https://www.gov.br/receitafederal/pt-br/acesso-a-informacao/acoes-e-programas/facilitacao/anexo-ecf-cfop)? If yes, maybe it'd be best to link the government page as well. Also, I didn't find the example value in this list.

Reply:
It is Brazilian only, I have updated the definition and example.